doc/source/v1/configuration.rst

```diff
@@ -19,6 +19,8 @@ The databroker looks for configuration files in three locations, in this order:
   virtual environment.)
 * ``/etc/databroker/``
 
+NOTE: For Windows, we only look in: ``%APPDATA%/databroker``.
```

Comment:
That seems...broken?

Reply:
Documents the code now with comment taken directly from the source code: https://github.com/bluesky/databroker/blob/2f7c3900301450021f0f79512ddd7d673c348345/databroker/utils.py#L237-L242

Reply:
fair enough!

Reply:
Should it be a backslash as a separator?

Reply:
For documentation purposes, that is right.  For python code, line 241 uses a different technique.  Since this was merged and the branch deleted, I will make the swap directly in master.  Ok?

Reply:
done

Reply:
Thank you, looks good.